roles/common/tasks/sshd.yml

```diff
@@ -0,0 +1,8 @@
+---
+
+- name: Disable root password login
```

Comment:
The SSH config is managed by cloud-init now - https://github.com/trailofbits/algo/tree/master/files/cloud-init. You can use either shell or cloud-config yaml, whatever Vultr supports

Reply:
I'll take a stab at this tomorrow.